src/ovp_pipeline/chat_fileops.py

```
@@ -649,6 +649,79 @@ def append_turn(
+
+    Used by the M22 drawer's Discard action — an operator who
+    finishes an inquiry without wanting to keep it in /search.
+    The session file lived as ``unindexed`` so nothing references
+    it; safe to unlink.
+
+    Missing target is not an error (idempotent discard).
+    """
+    try:
+        path.unlink()
+    except FileNotFoundError:
+        pass
```

Ruff (0.15.12)

[warning] Use `contextlib.suppress(FileNotFoundError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(FileNotFoundError): ...`

(SIM105)